### ISSUE TITLE
FR: REST API: /rest/release/* endpoints -- add more info

### ISSUE THREAD

**Issue by ramiromagno** [PGScatalog/PGS_Catalog#161]
Hi PGS Catalog team,

perhaps it would be nice to have a bit more information about releases:

- Right now I use the release `date` as a kind of key but this won't work if you make more than one release in one day. So perhaps returning a timestamp with resolution to the second would be best.
- It would also be nice to return the versioning of the Catalog database and the REST API itself. Then I could add two more columns to my first table (see below) in R: e.g., `catalog_version` and `rest_server_version`.

Currently this is how I parse your releases endpoints in R:

```R
An object of class "releases"                                                                                                                                              
Slot "releases":
# A tibble: 23 x 5
   date       n_pgs n_ppm n_pgp notes                                                                                         
   <date>     <int> <int> <int> <chr>                                                                                         
 1 2021-02-03    58   265     8 This release contains 58 new Score(s), 8 new Publication(s) and 265 new Performance metric(s) 
 2 2021-01-07     6    31     6 This release contains 6 new Score(s), 6 new Publication(s) and 31 new Performance metric(s)   
 3 2020-12-15   306   313     4 This release contains 306 new Score(s), 4 new Publication(s) and 313 new Performance metric(s)
 4 2020-12-08     9    65     6 This release contains 9 new Score(s), 6 new Publication(s) and 65 new Performance metric(s)   
 5 2020-11-20     4    50     5 This release contains 4 new Score(s), 5 new Publication(s) and 50 new Performance metric(s)   
 6 2020-11-05     4    19     4 This release contains 4 new Score(s), 4 new Publication(s) and 19 new Performance metric(s)   
 7 2020-10-19    79    79     1 This release contains 79 new Score(s), 1 new Publication(s) and 79 new Performance metric(s)  
 8 2020-10-16     1     1     1 This release contains 1 new Score(s), 1 new Publication(s) and 1 new Performance metric(s)    
 9 2020-09-18    10    34     3 This release contains 10 new Score(s), 3 new Publication(s) and 34 new Performance metric(s)  
10 2020-09-04     6    17     3 This release contains 6 new Score(s), 3 new Publication(s) and 17 new Performance metric(s)   
# … with 13 more rows

Slot "pgs_ids":
# A tibble: 721 x 2
   date       pgs_id   
   <date>     <chr>    
 1 2021-02-03 PGS000668
 2 2021-02-03 PGS000669
 3 2021-02-03 PGS000670
 4 2021-02-03 PGS000671
 5 2021-02-03 PGS000672
 6 2021-02-03 PGS000673
 7 2021-02-03 PGS000674
 8 2021-02-03 PGS000675
 9 2021-02-03 PGS000676
10 2021-02-03 PGS000677
# … with 711 more rows

Slot "ppm_ids":
# A tibble: 1,533 x 2
   date       ppm_id   
   <date>     <chr>    
 1 2021-02-03 PPM001396
 2 2021-02-03 PPM001397
 3 2021-02-03 PPM001398
 4 2021-02-03 PPM001399
 5 2021-02-03 PPM001400
 6 2021-02-03 PPM001401
 7 2021-02-03 PPM001402
 8 2021-02-03 PPM001403
 9 2021-02-03 PPM001404
10 2021-02-03 PPM001405
# … with 1,523 more rows

Slot "pgp_ids":
# A tibble: 133 x 2
   date       pgp_id   
   <date>     <chr>    
 1 2021-02-03 PGP000128
 2 2021-02-03 PGP000129
 3 2021-02-03 PGP000130
 4 2021-02-03 PGP000132
 5 2021-02-03 PGP000133
 6 2021-02-03 PGP000134
 7 2021-02-03 PGP000135
 8 2021-02-03 PGP000136
 9 2021-01-07 PGP000122
10 2021-01-07 PGP000123
# … with 123 more rows

```

**Comment by smlmbrt** [comment]
- We currently release data approximately every 3 weeks - it would be irregular for us to release twice in a day, so I think we'll pass on the timestamp.
- We will explore exposing the version and current release date in the API.

**Comment by ramiromagno** [comment]
Okay, thanks.

**Comment by ens-lgil** [comment]
Dear @ramiromagno,

For the REST API version, I was just thinking about adding it to the REST API.

My first thought would be to create its own endpoint (e.g. `/rest/info`) instead of adding a new field in the `/rest/release` endpoint: the REST API updates can be release (and data release) independent.

Best regards,
Laurent

**Comment by ramiromagno** [comment]
Thank you Laurent for your explanation.

Prior to your explanation, I thought that the `/rest/release` endpoint would pertain to any type of change made to REST API, be it data-related or REST API related. That is why I was suggesting add those two fields to help clarify what had changed.

It's clear now that `/rest/release` is only about data in the Catalog and that a putative `/rest/info` will be only about the REST API infrastructure. Cheers!

**Comment by ens-lgil** [comment]
Dear @ramiromagno,
  
A new endpoint `/rest/info` has just been released with the new REST API 1.6

Best regards,
Laurent
